Pipfile

```diff
@@ -22,6 +22,7 @@ uamqp = "*"
 requests = {extras = ["security"], version = "<2.32"}
 psycopg2 = "*"
 resolvelib = "==0.8.1"
+urllib3 = "*"
 
 [requires]
-python_version = "3.8"
+python_version = "3.10"
```

Comment:
This change is already proposed here: https://github.com/fjelltopp/fjelltopp-ansible/pull/36